tests/molecules/LiH_dimer_pp/det_vmc_hf_noj.in.xml

```diff
@@ -0,0 +1,2451 @@
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   9.750000000000e-04
+                      0.000000000000e+00  -1.308000000000e-03   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   7.731890000000e-01   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00  -4.400610000000e-01
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   8.509540000000e-01
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   6.040000000000e-03   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00  -2.542100000000e-02
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   6.862000000000e-03
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00   0.000000000000e+00   0.000000000000e+00   0.000000000000e+00
+                      0.000000000000e+00
```

Comment:
Remove samples here.  This will probably require that the reference data be regenerated.